src/main/kotlin/it/pagopa/wallet/services/MigrationService.kt

```diff
@@ -57,13 +57,14 @@ class MigrationService(
         userId: UserId
     ): Mono<Wallet> {
         logger.info(
-            "Initialize wallet for PaymentManager Id $paymentManagerWalletId and userId: ${userId.id}"
+            "Initialize wallet for paymentManagerId: $paymentManagerWalletId and userId: ${userId.id}"
```

Comment:
```suggestion
            "Initialize wallet for paymentManagerId: [{}] and userId: [{}]", paymentManagerWalletId, userId.id
```
Please use square brackets to contain value to be logged + avoid interpolation, but use the one provided by slf4j (this can avoid an useless string interpolation in case this logging level is shutdown, f.e. if we decide to log only ERROR or above levels for some reason)